package-lock.json

```diff
@@ -4124,9 +4124,9 @@
       }
     },
     "node_modules/minimist": {
-      "version": "1.2.5",
-      "resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.5.tgz",
-      "integrity": "sha512-FM9nNUYrRBAELZQT3xeZQ7fmMOBg6nWNmJKTcgsJeaLstP/UODVpGsr5OhXhhXg6f+qtJ8uiZ+PUxkDWcgIXLw==",
+      "version": "1.2.6",
```

Comment:
I don't think that `package-lock` should be modified within this PR. Please use `npm ci` command to install dependencies.

Reply:
I explicitly ran `npm audit fix` for soft fixing of vulnerabilities. Now there are 0 vulnerabilities.
There's separate commit for that.

Reply:
aha, got it!